API/resources/resources.json

```diff
@@ -52,10 +57,14 @@
           "dst": "%{tizenrt}/build/configs/artik053/jerryscript/"
         },
         {
-          "condition": "'%{app}' == 'jerryscript'",
+          "condition": "'%{app}' == 'jerryscript' and '%{build-type}' == 'release'",
           "src": "%{config}/tizenrt-jerryscript.config",
```

Comment:
Ditto. Please rename the `tizenrt-jerryscript.config` to `tizenrt-jerryscript-release.config`.